libmscore/shadownote.cpp

```diff
@@ -86,6 +86,15 @@ SymId ShadowNote::getNoteFlag() const
             case TDuration::DurationType::V_128TH:
                   flag = computeUp() ? SymId::flag128thUp : SymId::flag128thDown;
                   break;
+            case TDuration::DurationType::V_256TH:
+                  flag = computeUp() ? SymId::flag256thUp : SymId::flag256thDown;
+                  break;
+            case TDuration::DurationType::V_512TH:
+                  flag = computeUp() ? SymId::flag512thUp : SymId::flag512thDown;
+                  break;
+            case TDuration::DurationType::V_1024TH:
+                  flag = computeUp() ? SymId::flag1024thUp : SymId::flag1024thDown;
+                  break;
```

Comment:
Those shadow notes (seen during note input) have way too long stems. No idea how to fix that.^
Those stems seem to depend on the Musical font used, with Bravura even longer durations than those added here have quite long stems, with Emmentaler just these added ones.